debug.log

```diff
@@ -1 +1,3 @@
 [1025/153315.735:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1125/135749.363:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1126/022433.900:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Why do we need this file,?

Reply:
Some log files are in git repository so that a reason

Reply:
after the demo, it must be moved to .gitignore (when admin part has no pr`s)